specification/cognitiveservices/OpenAI.Inference/routes.tsp

```diff
@@ -99,3 +101,23 @@ op beginAzureBatchImageGeneration is OaiLongRunningRpcOperation<
   BatchImageGenerationOperationResponse,
   BatchImageGenerationOperationResponse
 >;
+
+@doc("Transcribes audio into the input language.")
+@added(ServiceApiVersions.v2023_09_01_Preview)
+@actionSeparator("/")
+@action("/deployments/{deployment-id}/audio/transcriptions") //@convenientAPI(true)
+op getTranscriptions is Azure.Core.ResourceAction<
+  Deployment,
+  TranscriptionsOptions,
+  Transcription
+>;
+
+@doc("Transcribes audio into the input language.")
```

Comment:
Copy/paste from transcription (but we can do a comments/docs pass across the whole thing anyway, as I think we can improve upon the source material)

Reply:
Thanks! And this made me look into the[ upstream json](https://github.com/Azure/azure-rest-api-specs-pr/pull/14569/files#diff-a6feb0a5a2dd571ac74fb27e7d8a5d88040b02e6a67bd5fa989e41a87aee3616R702) where the comment is `Transcribes and translates input audio into English text.` . Do we really only translate into English?